overlord/devicestate/devicemgr.go

```diff
@@ -149,6 +163,37 @@ func maybeReadModeenv() (*boot.Modeenv, error) {
 	return modeEnv, nil
 }
 
+func maybeSetupUbuntuSave() error {
+	saveMounted, err := osutil.IsMounted(dirs.SnapSaveDir)
+	if err != nil {
+		return err
+	}
+	if saveMounted {
+		logger.Noticef("save already mounted under %v", dirs.SnapSaveDir)
+		return nil
+	}
+
+	runMntSaveMounted, err := osutil.IsMounted(boot.InitramfsUbuntuSaveDir)
+	if err != nil {
+		return err
+	}
+	if !runMntSaveMounted {
+		// we don't have ubuntu-save
+		logger.Noticef("no ubuntu-save mount")
```

Comment:
this log is a bit confusing for UC16/18 where we don't plan to have save in any form

Reply:
maybe log it only if modeenv exists?

Reply:
perhaps this comment was about a prior state of the PR, but as it exists now, this is only called for UC20 which seems to be the right thing

Reply:
yes, this was previous to that change